deployment/ingress/eks/fuel-core-ingress.yaml

```diff
@@ -12,6 +12,12 @@ metadata:
     nginx.ingress.kubernetes.io/affinity-mode: persistent
     nginx.ingress.kubernetes.io/session-cookie-expires: "172800"
     nginx.ingress.kubernetes.io/session-cookie-max-age: "172800"
+    nginx.ingress.kubernetes.io/proxy-connect-timeout: "3600s"
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "3600s"
```

Comment:
Hmm, I'm afraid that it could cause a huge queue of requests. I think it worth to try it, but we need to monitor the queue size somehow=)

Reply:
it looks like we should be able to monitor this in prometheus, using the nginx exporter plugin: https://github.com/nginxinc/nginx-prometheus-exporter

Will make a ticket for this in the devops backlog

Reply:
https://github.com/FuelLabs/infrastructure/issues/99